authors/apps/tests/test_article.py

```diff
@@ -0,0 +1,55 @@
+from django.test import TestCase
+# from ..models import Article
```

Comment:
Since you created the articles model why do we still have this?

Reply:
Hello Rose, I am still cooking something, my next push will not have these errors. Please bear with me.